deps/make.blosc

```diff
@@ -50,7 +53,7 @@ CPPFLAGS += -DSHUFFLE_AVX2_ENABLED -DSHUFFLE_SSE2_ENABLED
 endif
 
 $(LIB): $(OBJ)
-	$(CC) $(CFLAGS) -shared $(OBJ) -o $(LIB)
+	$(CC) $(CFLAGS) -shared $(OBJ) -lstdc++ -o $(LIB)
```

Comment:
@stevengj I think this is the right way to link the C++ standard library, but I'm not super familiar with all of this stuff...

And yes, I'm also worried about the cross-compilation stuff and compat of that C++ standard library with the one that is used by julia itself...